.github/actions/npm-blacklist/checker.sh

```
@@ -109,3 +157,6 @@ else
     echo "✅ No vulnerable package versions found."
     exit 0
 fi
+
+# Cleanup
+rm -f "$dep_tree_file" "$blacklist_file_temp" "$results_file"
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)